apps/AppWithWearable/lib/utils/ble/connect.dart

```diff
@@ -18,6 +19,15 @@ Future<void> bleConnectDevice(String deviceId, {bool autoConnect = true}) async
       int desiredMtu = 512; // Example MTU size
       await device.requestMtu(desiredMtu);
     }
+    // Step 4: Monitor the connection state for reconnection
```

Comment:
Is this specific to Android? I didn't see this needed in iOS, could you check?

If that's the case, let's better try to configure the requestMtu and check the library for setting up autoConnect (which is reconnect). otherwise let's make it only for android

Reply:
@subhashdasyam